src/components/JSONLoader/JSONLoaderNav.tsx

```diff
@@ -10,24 +10,26 @@ export function LoaderNav({ loaderType: selectedLoaderType, onChange }: Props) {
   return (
     <nav>
       <ul>
-        {Object.values(LoaderType).map((loaderType) => (
-          <li key={loaderType}>
-            <button
-              onClick={() => onChange(loaderType)}
-              className={cn({
-                secondary: loaderType !== selectedLoaderType,
-              })}
-              data-tooltip={
-                loaderType === LoaderType.FromClipboard
-                  ? 'Recommend for huge JSON!'
-                  : undefined
-              }
-              data-placement="bottom"
-            >
-              {loaderType}
-            </button>
-          </li>
-        ))}
+        {Object.values(LoaderType)
+          .filter((loaderType) => loaderType !== LoaderType.FromURL)
```

Comment:
Exclude `LoaderType.FromURL` so that it can't be selected manually. But I'll consider more better way to handle it